tests/unit/bash_autocomplete_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+include './tests/unit/utils.sh'
+
+# Get and sort Bash autocomplete options
+function get_sorted_bash_completions() {
+  local completion='./src/bash_autocomplete.sh'
+  local cmd="$1"
+
+  source "${completion}"
+
+  COMP_WORDS=($cmd)
+  COMP_CWORD=1
+  COMPREPLY=()
```

Comment:
Could you add a brief comment above each of these variables that describes their purpose?